### DISCUSSION TITLE
can we get a "force sync" function?

### DISCUSSION THREAD

**Discussion by schwerd** [samschott/maestral#937] [General]
there are times when I've saved files from a working session, and it's time to close the laptop and get on a plane, or run to a meeting.  and as such, the files don't sync.  and then you expect to see them on the other machine and they aren't there. most times for hours but often for days.

it would be really useful in the menubar to have a "Sync Now" function, with a progress bar so that we can see that the sync is going and it's about to finish and then we can close the laptop and leave.

this seems to me a very very useful function.

thank you

**Comment by samschott** [comment]
Hi @schwerd, the files not syncing appears to be bug. In principle, every pause and resume or restart of Maestral will force a sync. Could you check if there is any pattern to the files that are affected? Or any pattern to when the delay in syncing occurs?

**Comment by schwerd** [comment]
I will do this, but it will take about a week to get on it. I’m traveling
and don’t have access to more than one machine for this.  But will check as
soon as I can.

**Comment by schwerd** [comment]
no pattern to files.  it seems eventually, things do sync.  sometimes happens quickly, sometimes takes an inordinate amount of time.  hard to gather any hard data or give examples as it's random and inconsistent.  I still think a "force sync" option would be really helpful.  especially with laptop and desktop. working remotely, saving files, then closing laptop and rushing off to catch plane or go to meeting.  being able to "force sync" before shutting down would solve a bunch of issues I have with files that I've saved and I think should be sync'd but aren't.
